src/NuGet.Core/NuGet.Packaging/PublicAPI/net472/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,3 @@
 #nullable enable
+~static NuGet.Packaging.Rules.AnalysisResources.MissingReadmeInformation.get -> string
```

Comment:
Can we keep this private for now? 

We're trying to make this experimental change, but by making it part of the API, we're committing too much imo.

Reply:
I checked our [coding-guideline](https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#when-to-use-internals-vs-public-and-when-to-use-internalsvisibleto), but it's not recommended to use `InternalsVisibleTo` in this case.
Shall I just add PackageCreationBestPracticeRules into PackageCreationRules in RuleSet then?

Reply:
Oh, I didn't notice this was a different assembly from the one where it's being used. 

I'm ambivalent about this. 

Curious what others think.

Reply:
So there are 3 options:
1. Keep the PackageCreationBestPracticeRuleSet.get a public API.
2. Change PackageCreationBestPracticeRuleSet into private, add PackageCreationBestPracticeRuleSet into PackageCreationBestPracticeRules.
3. Change the PackageCreationBestPracticeRuleSet.get into internal, and use ` InternalsVisibleTo`, which violates our [coding-guideline](https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#when-to-use-internals-vs-public-and-when-to-use-internalsvisibleto)
`InternalsVisibleTo is used only to allow a unit test to test internal types and members of its runtime assembly. We do not use InternalsVisibleTo between two runtime assemblies.`

If 2 is better, then when people who calls into PackageCreationBestPracticeRuleSet.get  API, they will get an extra item from the collection. Is that acceptable?

Reply:
I can't imagine too many people are running the validations. 

You're right, 3 is not really an option. 
I don't see a problem with 2.

Reply:
Thanks! Applied option 2.